### ISSUE TITLE
Use same binding again

### ISSUE THREAD

**Issue by vatsalnarry2** [opentelecoms-org/jsmpp#197]
hi ,

We are binding each time to smsc server and send an sms, when more transaction are coming from consumer, java application throw negative bind error.
I want to know is there any way to use existing connection instead of  using conenctAndBind() method repeatedly

**Comment by pmoerenhout** [comment]
Yes, this is the preferred method. You should do some link management, and re-use an existing connection.
